src/main/java/com/epam/ta/reportportal/entity/project/ProjectAttribute.java

```diff
@@ -0,0 +1,64 @@
+package com.epam.ta.reportportal.entity.project;
+
+import com.epam.ta.reportportal.entity.Attribute.Attribute;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+/**
+ * @author Andrey Plisunov
+ */
+@Entity
+@Table(name = "project_attribute", uniqueConstraints = @UniqueConstraint(columnNames = {"attribute_id", "project_id"}))
+public class ProjectAttribute implements Serializable {
```

Comment:
As i see in our database scheme, there is a table named 'attribute_group' but i don't see it's java object. Do we need it here?

Reply:
There shouldn't be `attribute_group` table. Please, double-check actual MR.